app/templates/index.html

```diff
@@ -135,31 +135,31 @@ <h4>Prediction Chart</h4>
 
                     </div>
                     <div id="annot-step-2" style="display: none">
-                      <img id="annot-img" />
-                      <div id="annot-sidebar">
-                      </div>
+                      <img id="annot-img" /><br />
                       Class: <input type="text" id="annot-class-drop" list="annot-classes">
                       <datalist id="annot-classes">
-                        <option value="1" />
-                        <option value="2" />
-                        <option value="3" />
-                        <option value="4" />
-                        <option value="5" />
-                        <option value="6" />
-                        <option value="7" />
-                        <option value="8" />
-                        <option value="9" />
-                        <option value="10" />
-                        <option value="11" />
-                        <option value="12" />
-                        <option value="13" />
-                        <option value="14" />
-                        <option value="15" />
-                        <option value="16" />
-                        <option value="17" />
-                        <option value="18" />
-                        <option value="19" />
-                        <option value="20" />
+                        <option value="0" >Prorocentrum micans</option>
```

Comment:
Did you verify with Sarah if these are the correct labels to enter?

Reply:
Yes, I took them straight from the document Melissa sent.